NFIQ2/nfiq2rf.txt

```diff
@@ -1,3 +1,4 @@
-Owner:    NIST/ISO
-Training: NFIQ2 Version 1.0
-Hash:     ccd75820b48c19f1645ef5e9c481c592
+Owner = NIST/ISO
```

Comment:
Let's clean this up some. Fields should be

- Name
- Description
- Version
- Model
- Hash